repos/msbuild.proj

```diff
@@ -2,25 +2,10 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <SolutionToBuild>$(ProjectDirectory)/NetCore.sln</SolutionToBuild>
-    <PackagesOutput>$(ProjectDirectory)/bin/$(Configuration)/packages</PackagesOutput>
-    <BuildCommand>$(DotnetToolCommand) pack $(SolutionToBuild) /p:Configuration=$(Configuration) /p:PackageOutputPath=$(PackagesOutput)</BuildCommand>
+    <PackagesOutput>$(ProjectDirectory)/artifacts/$(Configuration)/packages</PackagesOutput>
+    <BuildCommand>$(ProjectDirectory)build/build$(ShellExtension) build -DotNetBuildFromSource -DotNetCoreSdkDir $(DotNetCliToolDir) -bootstraponly -skiptests -pack -configuration $(Configuration) /p:DisableNerdbankVersioning=true /p:GitHeadSha=71eedf98975925fa7f2f3e669d19ba6c81391048</BuildCommand>
```

Comment:
I think the hard-coded commit hash should be stored in a property. (I'll have to do this for the auto-updater at some point anyway.) Unfortunately, we aren't standardized on `LatestCommit` or `GitCommitHash`, but more use `LatestCommit`.

Reply:
Agreed.  I thought it would be done as part of the updater task.

Reply:
Sounds good, just hard to spot at the end of a long line.

Reply:
I updated it so it doesn't get missed.